frontend/src/assets/manual/en-GB/word-embeddings.md

```diff
@@ -0,0 +1,57 @@
+
+### Finding related search terms
+
+When you start your research on a corpus, you often have a few keywords that you are interested in. For example, say that you want to search for *democracy*. A possible pitfall is that an author or speaker may talk about issues of democracy and representation, but not use this specific word. Since those documents will not match your search, you may never realise what you are missing.
+
+To help you out, you can use the word embeddings of a corpus to find terms related to your initial search term, *democracy*. When you explore the most similar words, you may find other terms such as *representation*, *democratic* or *parliament*. Perhaps you also find words that you would not have thought of, such as *liberalism*, which was very closely associated with the concept of democracy in the 19th century, but less so today.
+
+You may also find terms that are not synonymous with democracy, but are closely related because they are often contrasted with democracy, such as *dictatorship*. So if you are researching how Dutch authors discuss democracy and representation, it may be useful to extend your search to "dictatorship".
+
+So by looking at the related words, you can identify other search terms that may be interesting for your research.
+
+### Discovering other senses of a word
+
+When you have selected a search term for your research, it is possible that this term also has other meanings that you did not think of. For example, there might be a historic use of the term that you were not aware of.
+
+By exploring the most similar words, you can identify words that are related to other uses of your search term. For example, when we search for *democracy*, we may find terms that are very closely related to "social democracy", such as *socialism*, *labour*, and *marxism*. If an author mentions *social democracy*, they are discussing a political movement, but not necessarily talking about democracy as a concept.
+
+It's good to be aware of phrases or other meanings that might interfere with your results. If these other meanings are very common (perhaps more common than what you were interested in), you can try to exclude them when you search through the corpus.
+
+### Showing changes over time
+
+Word embeddings on I-analyzer are always *diachronic*: rather than creating a single model of the whole corpus, we train separate models for different time intervals. So when we look at the similarity scores between words, we can compare how that similarity changes over time.
+
+Changes in similarity can reflect changes in how words are used over time. Words can be used in new contexts and take on new meanings. (Or conversely, some usage may fall out of use.) Because such changes tend to cover long time periods, it can be difficult to reliably detect such changes if you are only relying on qualitative analysis. After all, you can only read a limited amount of text.
```

Comment:
(Or conversely, some *meanings* may fall out of use.) -> avoid repetition of "usage"/"use"